src/asciidoc/docs/refs/glossary.adoc

```diff
@@ -0,0 +1,33 @@
+= Glossary
+:page-slug: glossary
+:page-description: Definition of terms
+// Settings:
+:experimental:
+// Refs:
+:includespath: ../_includes
+
+entitlement::
+Configuration settings that allow the template or Koji to use certain platform features, such as enabling remixing or defining product information for in-app purchases.
+
+in-app purchase (IAP)::
+A financial transaction made within a Koji.
+
+Koji::
+A mini web application that can be shared on all social platforms, embedded in any website, and sent via messengers and email.
+Internally, a Koji is comprised of the associated template and the custom remix settings.
+Learn more: <<introduction#>>
+
+player chrome::
+The Koji button on a user's window.
```

Comment:
"user's window" doesn't seem quite accurate. It's really displayed on the Koji itself, or maybe "in the Koji player" would be the most accurate.

Reply:
Expanded the description, but it needs to be reviewed.